src/cryptography/x509/__init__.py

```diff
@@ -29,9 +29,9 @@
 )
 from cryptography.x509.name import Name, NameAttribute
 from cryptography.x509.oid import (
-    AuthorityInformationAccessOID, CRLExtensionOID, CertificatePoliciesOID,
-    ExtendedKeyUsageOID, ExtensionOID, NameOID, ObjectIdentifier,
-    SignatureAlgorithmOID, _SIG_OIDS_TO_HASH
+    AuthorityInformationAccessOID, CRLEntryExtensionOID,
```

Comment:
Yeah, if we're going to go to the effort to deprecate it, we should make
sure it's undistrubed.

On Fri, Dec 25, 2015 at 7:11 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/cryptography/x509/__init__.py
> <https://github.com/pyca/cryptography/pull/2574#discussion_r48443894>:
>
> > @@ -29,9 +29,9 @@
> >  )
> >  from cryptography.x509.name import Name, NameAttribute
> >  from cryptography.x509.oid import (
> > -    AuthorityInformationAccessOID, CRLExtensionOID, CertificatePoliciesOID,
> > -    ExtendedKeyUsageOID, ExtensionOID, NameOID, ObjectIdentifier,
> > -    SignatureAlgorithmOID, _SIG_OIDS_TO_HASH
> > +    AuthorityInformationAccessOID, CRLEntryExtensionOID,
>
> We never documented it, but yeah we should. Fixing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2574/files#r48443894>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084